src/Components/CardContainer.js

```diff
@@ -22,9 +27,9 @@ class CardContainer extends Component {
     render() {
       return(
           <div>
-           <ul>
+           <section>
```

Comment:
I agree with this, it makes more sense to have the cards in a section rather than an unordered list.